<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QDialog" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reception</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../app/icons.qrc">
    <normaloff>:/warehouse</normaloff>:/warehouse</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>931</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>DEFINE DEVICE:</string>
   </property>
   <widget class="QLineEdit" name="sn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="actual_item">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="actual_condition">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="actual_spec">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SN/IMEI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>17</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUAL ITEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>623</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUAL CONDITION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>23</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUAL SPEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>32</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COMMIT</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="imei_check">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>66</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Valid IMEI check</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>731</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>DEFINED DEVICES SUMMARY:</string>
   </property>
   <widget class="ClipView" name="view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>731</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>510</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>SN/IMEIS:</string>
   </property>
   <widget class="ClipListView" name="snlist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>931</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>GENERAL</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Processed:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reception_total">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>128</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>69</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Agent:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="agent">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>74</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>42</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Partner:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="warehouse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>101</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="partner">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>47</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>381</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>381</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="reception_number">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="date">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_processed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>155</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>96</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Warehouse:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>123</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>15</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Order Nº:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lines_group">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>931</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>MIXED/NORMAL LINES:</string>
   </property>
   <widget class="QLabel" name="Label2_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Condition:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="condition">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="spec">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>0</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="next">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="icon">
     <iconset resource="../app/icons.qrc">
      <normaloff>:/right_arrow</normaloff>:/right_arrow</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>184</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Label10">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>processed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="processed">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>22</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="line_number">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label2_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Spec:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="prev">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
    <property name="icon">
     <iconset resource="../app/icons.qrc">
      <normaloff>:/cancel</normaloff>:/cancel</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>185</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Label8">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Line Total :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_total">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nº</string>
    </property>
   </widget>
   <widget class="QLabel" name="overflow">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:&quot;bold&quot;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>860</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_line_edit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>480</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="all">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>860</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="automatic">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>370</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>AUTOMATIC INPUT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMEI/SN LENGHT:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lenght">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/exit</normaloff>:/exit</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClipListView</class>
   <extends>QListView</extends>
   <header location="global">clipview</header>
  </customwidget>
  <customwidget>
   <class>ClipView</class>
   <extends>QTableView</extends>
   <header location="global">clipview</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sn</tabstop>
  <tabstop>actual_item</tabstop>
  <tabstop>actual_condition</tabstop>
  <tabstop>actual_spec</tabstop>
  <tabstop>all</tabstop>
  <tabstop>automatic</tabstop>
  <tabstop>delete_button</tabstop>
  <tabstop>search_line_edit</tabstop>
  <tabstop>search</tabstop>
  <tabstop>commit</tabstop>
  <tabstop>view</tabstop>
  <tabstop>snlist</tabstop>
  <tabstop>reception_total</tabstop>
  <tabstop>agent</tabstop>
  <tabstop>warehouse</tabstop>
  <tabstop>partner</tabstop>
  <tabstop>reception_number</tabstop>
  <tabstop>date</tabstop>
  <tabstop>total_processed</tabstop>
  <tabstop>condition</tabstop>
  <tabstop>description</tabstop>
  <tabstop>spec</tabstop>
  <tabstop>next</tabstop>
  <tabstop>line_number</tabstop>
  <tabstop>prev</tabstop>
  <tabstop>lenght</tabstop>
  <tabstop>processed</tabstop>
  <tabstop>line_total</tabstop>
 </tabstops>
 <resources>
  <include location="../app/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1175</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>1162</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
